Dockerfile.alpine

```diff
@@ -1,5 +1,5 @@
-FROM alpine:latest
-MAINTAINER Matt Bentley <mbentley@mbentley.net>
+FROM ghcr.io/linuxserver/baseimage-alpine-nginx:3.13
```

Comment:
My issues with changing the base image are that I don't know what all is in that base image, I am not sure how they update it, or when the update it.